client/site-profiler/hooks/use-delayed-query-time.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
+
+// Used to generate a different text for the "Check site" button
+export function useDelayedQueryTime( domain: string, isFetching: boolean, limit = 3000 ) {
```

Comment:
Please export it as a **default** function to follow the convention between hooks.

Reply:
Fixed.